lib/run/summary.js

```diff
@@ -38,6 +38,16 @@ RunSummary = function RunSummary (emitter, options) {
          */
         globals: _.get(options, 'globals'),
 
+        /**
+         * Holds information related to the reporters specified.
+         */
+        reporters: _.get(options, 'reporters', []),
+
+        /**
+         * Holds information related to skip addition of response to the execution report generated.
+         */
+        skipExecutionResponse: _.get(options, 'skipExecutionResponse'),
+
```

Comment:
Do we need to add the `reporters` and `skipExecutionResponse` options to the `RunSummary` object? This would change the API, no? Can we not just use them wherever required ad-hoc?